org.eclipse.jdt.text.tests/src/org/eclipse/jdt/text/tests/templates/TemplateCompletionTests.java

```diff
@@ -102,9 +101,7 @@ public void testExepectNoProposals() throws Exception {
 		List<ICompletionProposal> proposals= computeCompletionProposals(cu, completionIndex);
 
 		boolean fail= proposals.stream().anyMatch(p -> "new_class - create new class".equals(p.getDisplayString()));
-		if (fail) {
-			fail("Proposal '" + propDisplay + "' should not exist");
-		}
+		org.junit.Assert.assertFalse("Proposal '" + propDisplay + "' should not exist", fail);
```

Comment:
Please import org.junit.Assert.assertFalse above like we do for assertEquals and assertTrue.  This comment applies to all the changes.

Reply:
I added some more files but missed a few imports. So in a few cases still something to fix unfortunately.

Reply:
I think I catched all now.